src/components/utils/__mocks__/mq.js

```diff
@@ -0,0 +1,114 @@
+// @flow strict
```

Comment:
Please let me express my doubts about this mock:
- I don't know if it's working correctly (we're testing this mock by our test which relies on it at the same time, this mock doesn't have dedicated tests to cover its behaviour)
- I don't know if it's working the same way as in browsers.

This is generic issue with any kind of mock but in our case we effectively couple mock and hook together.

This is not a blocker but rather something we need to consider more carefully in the future. Anyway thanks for covering at least happy path.

Reply:
I share your opinion

- I was considering writing test for this mock but I have doubts to the approach itself
- Yes because it is a monkey patching 

But this mock helps to make sure that at leas that hook itself along with listeners are working correctly.

I think the best solution to test this util will be E2E cypress test thus we can force "prefers-reduced-motion" and relay on it.  As we do not have cypress we need to test story manually